File-Formats/PDF/Convert-HTML-To-PDF/azure.md

```diff
@@ -77,375 +78,332 @@ private void InstallDependecies(string shellFilePath)
       process.WaitForExit();
 }
 
-{% endhighlight %}
+   {% endhighlight %}
 
-Add an Export to the PDF button in the index.cshtml.
+   Add an Export to the PDF button in the index.cshtml.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-<div class="btn">
+   <div class="btn">
     @{ Html.BeginForm("ExportToPDF", "Home", FormMethod.Post);
         {
             <input type="submit" value="Export To PDF" class=" btn" />
         }
-}
-</div>
-
-{% endhighlight %}
-
-Include the following namespaces and code samples in the controller for converting HTML to PDF.
-
-{% highlight c# tabtitle="C#" %}
-
-using Syncfusion.HtmlConverter;
-using Syncfusion.Pdf;
-using System.IO
-
-{% endhighlight %}
-
-{% highlight c# tabtitle="C#" %}
-
-public ActionResult ExportToPDF()
-{
-     Environment.SetEnvironmentVariable("ASPNETCORE_ENVIRONMENT", "Development"); 
+   }
+   </div>
 
-	//Initialize HTML to PDF converter 
-	HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+   {% endhighlight %}
 
-	BlinkConverterSettings settings = new BlinkConverterSettings();
+   Include the following namespaces and code samples in the controller for converting HTML to PDF.
```

Comment:
Add numbering to this step for add namespace and add another step with numbering for add code sample.

Reply:
Added separate for namespace and code